packages/trader/src/AppV2/Containers/Trade/__tests__/trade.spec.tsx

```diff
@@ -156,7 +156,6 @@ describe('Trade', () => {
         expect(screen.getByText('MarketSelector')).toBeInTheDocument();
         expect(screen.getAllByText('Trade Parameters')).toHaveLength(2);
         expect(screen.getByText('Chart')).toBeInTheDocument();
-        expect(screen.getByText('Purchase Button')).toBeInTheDocument();
```

Comment:
Sorry, just for my understanding: why purchase button now should not be rendered on trading page?

Reply:
actually i was doing something else before which removed purchase button. Adding the check back.